website/pages/style.css

```diff
@@ -10,15 +10,15 @@
 
 @import '~@hashicorp/react-button/styles/index.css';
 @import '~@hashicorp/react-code-block/style.css';
-@import '~@hashicorp/react-consent-manager/dist/style.css';
+@import '~@hashicorp/react-consent-manager/style.css';
 @import '~@hashicorp/react-content/style.css';
 @import '~@hashicorp/react-docs-page/style.css';
 @import '~@hashicorp/react-docs-sidenav/style.css';
 @import '~@hashicorp/react-search/style.css';
 @import '~@hashicorp/react-section-header/style.css';
 @import '~@hashicorp/react-subnav/style.css';
-@import '~@hashicorp/react-tabs/dist/style.css';
-@import '~@hashicorp/react-toggle/dist/style.css';
+@import '~@hashicorp/react-tabs/style.css';
+@import '~@hashicorp/react-toggle/style.css';
```

Comment:
Updated some imports, per [the helpful `nextjs-scripts@14.0.0` release notes](https://github.com/hashicorp/nextjs-scripts/releases/tag/14.0.0)